tests/integration/setup.ts

```diff
@@ -68,6 +68,19 @@ const buildCredentials = function () {
 			username: env.TEST_REGISTER_USERNAME,
 		},
 	};
+	if (
+		![creds.email, creds.paid.email, creds.register.email].every((email) => {
+			if (email) {
+				return email.includes('+testsdk');
+			} else {
+				return true;
+			}
+		})
```

Comment:
```suggestion
		![creds.email, creds.paid.email, creds.register.email].every((email) => email == null || email.includes('+testsdk'))
```

Also, can we leave the paid user tests out? (I would need to reach to the customer success for this, which would take time, and we don't do any write operations for that user.)

Reply:
how about i leave it out for now, but we start the process of updating that as well? even though we do no writing now, that may change in the future and unless we enforce it as an invariant i worry about future data loss.

Reply:
Sure

Reply:
thank you for all your help :heart: